[2025-09-13T17:55:51.277+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-13T17:55:51.341+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_etl.send_forecast scheduled__2025-09-12T16:00:00+00:00 [queued]>
[2025-09-13T17:55:51.357+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_etl.send_forecast scheduled__2025-09-12T16:00:00+00:00 [queued]>
[2025-09-13T17:55:51.358+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-09-13T17:55:51.380+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): send_forecast> on 2025-09-12 16:00:00+00:00
[2025-09-13T17:55:51.396+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'currency_etl', 'send_forecast', 'scheduled__2025-09-12T16:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/currency_etl.py', '--cfg-path', '/tmp/tmpk_to4w7m']
[2025-09-13T17:55:51.401+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1289) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-13T17:55:51.402+0000] {standard_task_runner.py:105} INFO - Job 252: Subtask send_forecast
[2025-09-13T17:55:51.403+0000] {standard_task_runner.py:72} INFO - Started process 1290 to run task
[2025-09-13T17:55:51.586+0000] {task_command.py:467} INFO - Running <TaskInstance: currency_etl.send_forecast scheduled__2025-09-12T16:00:00+00:00 [running]> on host 6eb00eb09733
[2025-09-13T17:55:51.852+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_etl' AIRFLOW_CTX_TASK_ID='send_forecast' AIRFLOW_CTX_EXECUTION_DATE='2025-09-12T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-12T16:00:00+00:00'
[2025-09-13T17:55:51.856+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-13T17:55:55.289+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-09-13T17:55:55.342+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-13T17:55:55.343+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=currency_etl, task_id=send_forecast, run_id=scheduled__2025-09-12T16:00:00+00:00, execution_date=20250912T160000, start_date=20250913T175551, end_date=20250913T175555
[2025-09-13T17:55:55.441+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-13T17:55:55.513+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-13T17:55:55.517+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
